### DISCUSSION TITLE
Auto stack option for Advenced Magnet Upgrade or Advanced Pickup Upgrade

### DISCUSSION THREAD

**Discussion by ValkyrieSama** [P3pp3rF1y/SophisticatedBackpacks#99] [Ideas]
It could be nice if we had an option to fill existing stack in the backpack when picking up items with the upgrades. As I saw, now it place the item on the first slot available or the first existing stack from the top row of the backpack.

**Comment by TheDeviantCrafter** [comment]
This should probably be the default behavior. I can't think of a situation where not stacking is helpful.

**Reply by P3pp3rF1y** [reply]
I agree that it's more intuitive, problem is how this works behind the scenes and how often it may need to be triggered by automation. So I will need to code around this quite a bit and make sure it's properly cached not to cause performance issues.
